ui/components/ErrorBoundary.js

```diff
@@ -1,41 +1,12 @@
-import { Button } from '@material-ui/core';
 import React from 'react';
+import { ErrorBoundary as SistentErrorBoundary } from '@layer5/sistent';
 
-/**
- * ErrorBoundary is a React component that catches JavaScript errors in its child components and renders a fallback UI when an error occurs.
- * It should be used as a wrapper around components that might throw errors.
- * @deprecated use error boundary from sistent instead
- */
-class ErrorBoundary extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = { hasError: false, error: null };
-  }
-
-  /** Update state so the next render will show the fallback UI. */
-  static getDerivedStateFromError(error) {
-    return { hasError: true, error: error };
-  }
-
-  resetErrorBoundary = () => {
-    this.setState({ hasError: false, error: null });
-  };
+function handleError(error) {
+  console.error('Error in Spaces Preferences Component', error);
+}
 
-  /** You can render any custom fallback UI */
-  render() {
-    if (this.state.hasError) {
-      return (
-        <div className="alert alert-danger">
-          <p>Couldn&apos;t open form. Encountered the following error:</p>
-          <pre>{this.state.error.message}</pre>
-          <Button color="primary" variant="contained" onClick={this.resetErrorBoundary}>
-            Refresh Form
-          </Button>
-        </div>
-      );
-    }
-    return this.props.children;
-  }
+function ErrorBoundary({ children }) {
+  return <SistentErrorBoundary onError={handleError}>{children}</SistentErrorBoundary>;
```

Comment:
Hey @THEBOSS0369 , i dont believe the sistentErrorboundary takes an onError handle. In this case, we can probably delete this file and use the error boundary directly vs adding another wrapper.

Reply:
Hey @ianrwhitney ! Yes i also think that this file is not needed because when i was importing it i saw it is already present there. So I'm Deleting this File.